src/java.net.http/share/classes/jdk/internal/net/http/frame/SettingsFrame.java

```diff
@@ -72,21 +71,16 @@ public String toString() {
     public static final int MAX_HEADER_LIST_SIZE = 0x6;
 
     private String name(int i) {
-        switch (i+1) {
-        case HEADER_TABLE_SIZE:
-            return "HEADER_TABLE_SIZE";
-        case ENABLE_PUSH:
-            return "ENABLE_PUSH";
-        case MAX_CONCURRENT_STREAMS:
-            return "MAX_CONCURRENT_STREAMS";
-        case INITIAL_WINDOW_SIZE:
-            return "INITIAL_WINDOW_SIZE";
-        case MAX_FRAME_SIZE:
-            return "MAX_FRAME_SIZE";
-        case MAX_HEADER_LIST_SIZE:
-            return "MAX_HEADER_LIST_SIZE";
-        }
-        return "unknown parameter";
+        return switch (i + 1) {
+            case HEADER_TABLE_SIZE ->       "HEADER_TABLE_SIZE";
+            case ENABLE_PUSH ->             "ENABLE_PUSH";
+            case MAX_CONCURRENT_STREAMS ->  "MAX_CONCURRENT_STREAMS";
+            case INITIAL_WINDOW_SIZE ->     "INITIAL_WINDOW_SIZE";
+            case MAX_FRAME_SIZE ->          "MAX_FRAME_SIZE";
+            case MAX_HEADER_LIST_SIZE ->    "MAX_HEADER_LIST_SIZE";
+            
+            default -> "unknown parameter";
```

Comment:
> Check failure on line 81 in src/java.net.http/share/classes/jdk/internal/net/http/frame/SettingsFrame.java
> openjdk / jcheck
> 
> Whitespace error
> 
> Column 0: trailing whitespace
> ...

WRT to whitespace errors detected by `jcheck`, note that you can fix them by running the script:

```
make/scripts/normalizer.pl
```

on this file.

Reply:
That's handy. Thanks!